charts/k8s-monitoring/templates/agent_config/_kubernetes_monitoring_telemetry.river.txt

```diff
@@ -0,0 +1,28 @@
+{{ define "agent.config.kubernetes_monitoring_telemetry" }}
+// Kubernetes Monitoring Telemetry
+prometheus.exporter.unix {
+  set_collectors = ["textfile"]
+  textfile {
+    directory = "/etc/kubernetes-monitoring-telemetry"
+  }
+}
+
+prometheus.scrape "kubernetes_monitoring_telemetry" {
+  job_name   = "integrations/kubernetes/kubernetes_monitoring_telemetry"
+  targets    = prometheus.exporter.unix.targets
+  scrape_interval = {{ .Values.metrics.scrapeInterval | quote }}
+{{- if (index .Values "grafana-agent").agent.clustering.enabled }}
+  clustering {
+    enabled = true
+  }
+{{- end }}
+  forward_to = [prometheus.relabel.kubernetes_monitoring_telemetry.receiver]
+}
+
+prometheus.relabel "kubernetes_monitoring_telemetry" {
+{{- if .Values.metrics.extraMetricRelabelingRules }}
+{{ .Values.metrics.extraMetricRelabelingRules | indent 2 }}
```

Comment:
Does it still make sense to allow relabelling here for `kubernetes_monitoring_telemetry`? I'm not sure of the use case for relabelling that metric, I can only think of ways it could break or be abused... 😅

Reply:
My thinking: This allows people to add custom labels if they wanted (e.g. `region="netherton"`)

Reply:
Ok, I see. Makes sense then 👍